.github/workflows/checks.yml

```diff
@@ -0,0 +1,40 @@
+name: Checks
```

Comment:
Looks like you're using GH actions for ci -- we typically use circleci in our other repos (for any meaningful testing). Not necessarily against using GH actions, but I think we should generally try to move towards using a single ci system. Maybe we can discuss in our next eng meeting whether we prefer GH actions or circleci!

Are there things you specifically prefer about GH actions that don't exist in other ci platforms?

Reply:
I went for GH since that's what I usually do for CI. I have no strong preference, and since scattering CI across services sounds like the last thing we want, I'm removing them from this PR until we settle on which CI to use.

Reply:
That sounds good for now. Let's definitely discuss the tradeoffs -- would be very interested to get your feedback on what you've found good and bad about GH actions since you probably have more experience on that platform than the rest of the team.